kubetest/gke_test.go

```diff
@@ -0,0 +1,180 @@
+					zone: "us-central1-a",
+					name: `gke-default-pool-6f8ec327-grp`,
+					uniq: `6f8ec327`,
+				},
+			},
+			expectedError: false,
+		},
+		{
+			name:                  "One instance group with default prefix, but not matching instance_group_prefix",
+			input:                 urlZoneADefault,
+			instance_group_prefix: "test",
+			expected:              nil,
+			expectedError:         true,
+		},
+		{
+			name:                  "One instance group with test prefix",
+			input:                 urlZoneATest,
+			instance_group_prefix: "test",
+			expected: []*ig{
+				{
+					path: `zones/us-central1-a/instanceGroupManagers/test-default-pool-6f8ec327-grp`,
+					zone: "us-central1-a",
+					name: `test-default-pool-6f8ec327-grp`,
+					uniq: `6f8ec327`,
+				},
+			},
+			expectedError: false,
+		},
+		// Regional
+		{
+			name:                  "Three instance groups with default prefix",
+			input:                 strings.Join([]string{urlZoneADefault, urlZoneBDefault, urlZoneCDefault}, ";"),
+			instance_group_prefix: "",
+			expected: []*ig{
+				{
+					path: `zones/us-central1-a/instanceGroupManagers/gke-default-pool-6f8ec327-grp`,
+					zone: "us-central1-a",
+					name: `gke-default-pool-6f8ec327-grp`,
+					uniq: `6f8ec327`,
+				},
+				{
+					path: `zones/us-central1-b/instanceGroupManagers/gke-default-pool-6f8ec327-grp`,
+					zone: "us-central1-b",
+					name: `gke-default-pool-6f8ec327-grp`,
+					uniq: `6f8ec327`,
+				},
+				{
+					path: `zones/us-central1-c/instanceGroupManagers/gke-default-pool-6f8ec327-grp`,
+					zone: "us-central1-c",
+					name: `gke-default-pool-6f8ec327-grp`,
+					uniq: `6f8ec327`,
+				},
+			},
+			expectedError: false,
+		},
+		{
+			name:                  "Three instance groups with default prefix, but not matching instance_group_prefix",
+			input:                 strings.Join([]string{urlZoneADefault, urlZoneBDefault, urlZoneCDefault}, ";"),
+			instance_group_prefix: "test",
+			expected:              nil,
+			expectedError:         true,
+		},
+		{
+			name:                  "Three instance groups with test prefix",
+			input:                 strings.Join([]string{urlZoneATest, urlZoneBTest, urlZoneCTest}, ";"),
+			instance_group_prefix: "test",
+			expected: []*ig{
+				{
+					path: `zones/us-central1-a/instanceGroupManagers/test-default-pool-6f8ec327-grp`,
+					zone: "us-central1-a",
+					name: `test-default-pool-6f8ec327-grp`,
+					uniq: `6f8ec327`,
+				},
+				{
+					path: `zones/us-central1-b/instanceGroupManagers/test-default-pool-6f8ec327-grp`,
+					zone: "us-central1-b",
+					name: `test-default-pool-6f8ec327-grp`,
+					uniq: `6f8ec327`,
+				},
+				{
+					path: `zones/us-central1-c/instanceGroupManagers/test-default-pool-6f8ec327-grp`,
+					zone: "us-central1-c",
+					name: `test-default-pool-6f8ec327-grp`,
+					uniq: `6f8ec327`,
+				},
+			},
+			expectedError: false,
+		},
+	}
+
+	for _, tc := range cases {
```

Comment:
having test case names, t.Run should be used with this name
https://blog.golang.org/subtests

Reply:
Done.